swagger/src/test/scala/me/apidoc/swagger/SwaggerServiceValidatorSpec.scala

```diff
@@ -47,6 +39,172 @@ class SwaggerServiceValidatorSpec extends FunSpec with Matchers {
+                      path = "/pets/",
+                      description = Some("finds pets by status - as a query param"),
+                      deprecation = None,
+                      body = None,
+                      parameters = Seq(Parameter(
+                        name = "status",
+                        `type` = "enum_status",
+                        location = Query,
+                        description = None,
+                        deprecation = None,
+                        required = true,
+                        default = None,
+                        minimum = None,
+                        maximum = None,
+                        example = None)
+                      ),
+                      responses = Seq(
+                        Response(
+                          code = ResponseCodeInt(200),
+                          `type` = "[Pet]",
+                          description = Some("find pet response"),
+                          deprecation = None),
+                        Response(
+                          code = ResponseCodeOption.Default,
+                          `type` = "Error",
+                          description = Some("unexpected error"),
+                          deprecation = None)
+                      ),
+                      attributes = Nil),
+                    Operation(
+                      method = Get,
+                      path = "/pets/:status",
+                      description = Some("finds pets by status - as a path param"),
+                      deprecation = None,
+                      body = None,
+                      parameters = Seq(Parameter(
+                        name = "status",
+                        `type` = "enum_status",
+                        location = Path,
+                        description = None,
+                        deprecation = None,
+                        required = true,
+                        default = None,
+                        minimum = None,
+                        maximum = None,
+                        example = None)
+                      ),
+                      responses = Seq(
+                        Response(
+                          code = ResponseCodeInt(200),
+                          `type` = "[Pet]",
+                          description = Some("find pet response"),
+                          deprecation = None),
+                        Response(
+                          code = ResponseCodeOption.Default,
+                          `type` = "Error",
+                          description = Some("unexpected error"),
+                          deprecation = None)
+                      ),
+                      attributes = Nil)
+                  ),
+                  attributes = Seq())
+                )
+
+              service.enums.size should be(1)
+              checkEnum(service.enums(0),
+                Enum(
+                  name = "enum_status",
```

Comment:
In chatting w/ group - we're hoping we can change the naming process to:

  - if there is no other types named 'status' - just use 'status' as the name of the enum
  - otherwise, try "PetStatus" where "Pet" is coming from the resource under which the operation is defined
  - otherwise, try "PetStatusGet" where "Get" is the path of the operation (and include other path variables similar to how we name client methods by default)